petab/problem.py

```diff
@@ -88,7 +88,8 @@ def __setstate__(self, state):
     @staticmethod
     def from_files(
             sbml_file: Union[str, Path, None] = None,
```

Comment:
```suggestion
            sbml_file: Union[str, Path] = None,
```
Not sure, why can `sbml_file` and `condition_file` be specified as `None` by the user, but not the other tables?

Reply:
All of them should be optional.